docs/index.rst

```diff
@@ -1,55 +1,49 @@
 
-F5's Container Integrations make it possible to dynamically allocate BIG-IP L4-L7 services in container orchestration environments. The Container Connectors ('CCs') understand the container orchestration environment ('COE'), thereby providing PaaS-native integrations for F5 BIG-IP devices.
+.. table:: F5 Container Connectors
 
-=======================     ===================================================
-Container Connector         Description
-=======================     ===================================================
-`cf-bigip-ctlr`_            Integrates BIG-IP as an
-                            Application Delivery Controller (ADC) in
-                            `Pivotal Cloud Foundry`_, serving North-South
-                            traffic.
------------------------     ---------------------------------------------------
-`k8s-bigip-ctlr`_           Configures a BIG-IP device to expose applications
-                            in `Kubernetes`_ and `Red Hat OpenShift`_ clusters
-                            as virtual servers, serving North-South traffic.
------------------------     ---------------------------------------------------
-`marathon-bigip-ctlr`_      Configures a BIG-IP device to expose applications
-                            in an `Apache Mesos`_ `Marathon`_ cluster as
-                            virtual servers, serving North-South traffic.
-=======================     ===================================================
+   =======================     ===================================================
+   Container Connector         Description
+   =======================     ===================================================
+   `cf-bigip-ctlr`_            Integrates BIG-IP as an
+                               Application Delivery Controller (ADC) in
+                               `Pivotal Cloud Foundry`_, serving North-South
+                               traffic.
+   -----------------------     ---------------------------------------------------
+   `k8s-bigip-ctlr`_           Configures a BIG-IP device to expose applications
+                               in `Kubernetes`_ and `Red Hat OpenShift`_ clusters
+                               as virtual servers, serving North-South traffic.
+   -----------------------     ---------------------------------------------------
+   `marathon-bigip-ctlr`_      Configures a BIG-IP device to expose applications
+                               in an `Apache Mesos`_ `Marathon`_ cluster as
+                               virtual servers, serving North-South traffic.
+   =======================     ===================================================
 
-\
+.. figure:: /_static/media/container_connectors_north-south.png
+   :scale: 80
```

Comment:
This might be a linux firefox browser issue, but I'm not seeing the image scale - have you viewed this on mac or windows?

Reply:
Fixed in the new version of the f5-sphinx-theme (coming soon!).